BookPlayer/Library/BaseListViewController.swift

```diff
@@ -81,6 +82,19 @@ class BaseListViewController: UIViewController {
         bookCell.playbackState = .paused
     }
 
+    @objc func onBookStop(_ notification: Notification) {
+        guard
+            let userInfo = notification.userInfo,
+            let book = userInfo["book"] as? Book,
+            let index = self.library.itemIndex(with: book.fileURL),
```

Comment:
This needs to be overridden in the `PlaylistViewController` to get the index inside the playlist

Reply:
Done.